spec/config/other_linker_flags_parser_spec.rb

```diff
@@ -123,12 +169,15 @@
       flags << '-lxml2.2.7.3'
       flags << '-l "Pods-AFNetworking iOS Example-AFNetworking"'
       flags << '-l"Pods-AFNetworking iOS Example-AFNetworking"'
+      flags << '@"${PODS_ROOT}/Target Support Files/other_ldflags"'
+      flags << '@ "${PODS_ROOT}/Target Support Files/other_ldflags"'
```

Comment:
is there a test case covering a space between `@` and the quoted file path?

Reply:
I believe so yes, right above

Reply:
thanks, I missed it